src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/Example5.java

```diff
@@ -0,0 +1,21 @@
+/*xml
+<module name="Checker">
+  <module name="RegexpSingleline">
+    <property name="format" value="^[ ]*\*This file is copyrighted"/>
+    <property name="minimum" value="1"/>
+    <!--  Need to specify a maximum, so 10 times is more than enough. -->
+    <property name="maximum" value="10"/>
+    <property name="message"
+                value="File must contain copyright statement"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.regexp.regexpsingleline;
+
+// xdoc section -- start
+/**
+ * // violation, as missing copyright
```

Comment:
As this is java file. Please move this line out of javadoc comment to be above javadoc comment.
It will be `// violation below `File must contain copyright statement'`

Reply:
> As this is java file. Please move this line out of javadoc comment to be above javadoc comment. It will be `// violation below `File must contain copyright statement'`

I tried this but still couldn't get the test to pass. In the end I used the same approach by [regexpheader ](https://github.com/checkstyle/checkstyle/blob/master/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/header/regexpheader/Example2.java).